source/dofs/dof_tools_sparsity.inst.in

```diff
@@ -257,15 +257,17 @@ for (deal_II_dimension : DIMENSIONS)
   }
 
 
-for (deal_II_dimension : DIMENSIONS)
+for (deal_II_dimension, deal_II_space_dimension : DIMENSIONS)
```

Comment:
```suggestion
for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS)
```
We have split dim and spacedim into two separate template argument lists in other occassions.

See also
https://github.com/dealii/dealii/blob/20d2176d317137ec1b5c8fc1334cfef649765c14/cmake/config/template-arguments.in#L254
https://github.com/dealii/dealii/blob/a65d7b464723ebdb141d1f648074e9a844342dc6/source/grid/tria.inst.in#L18

Reply:
Hmm, we wrote `(deal_II_dimension, deal_II_space_dimension : DIMENSIONS)` also in other parts of the library. Maybe it's better to discuss it in a separate PR. I will prepare one.

Reply:
@marcfehling Yes, let's fix that up in a separate PR so that we catch all occurences.